### PR TITLE
Small fixes found in production

### PR DESCRIPTION
**Pull request by Edholm**

### CODE REVIEW

File: backend/src/main/kotlin/rocks/didit/sefilm/domain/Swish.kt [hunk 1]
```diff
@@ -23,7 +23,7 @@ data class SwishDataDTO(
 
     fun generateUri(): URI {
         val asString = jacksonObjectMapper().writeValueAsString(this)
-        val encodedData = URLEncoder.encode(asString, StandardCharsets.UTF_8.displayName())
+        val encodedData = URLEncoder.encode(asString, StandardCharsets.UTF_8.displayName()).replace("+", "%20")
```
Comment (lindskogen) [inline backend/src/main/kotlin/rocks/didit/sefilm/domain/Swish.kt]: Can we find a solution that doesn't remove actual plus signs?

File: frontend/src/CopyValue.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ class CopyValue extends Component {
   }
 
   render() {
-    return <Hover onClick={this.onClick}>{this.props.text}</Hover>
+      return <div><Hover onClick={this.onClick}>{this.props.text}</Hover></div>
```
Comment (lindskogen) [inline frontend/src/CopyValue.js]: We should find a better solution that this, but it's fine for now!

File: frontend/src/routes/SingleShowing.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ const UserActiveStatus = styled.div`
 `
 
 const UserItem = buildUserComponent(({ user }) => (
-  <span>{user.nick || user.name} ({user.phone})</span>
+  <div>{user.nick || user.name} ({user.phone})</div>
```
Comment (lindskogen) [inline frontend/src/routes/SingleShowing.js]: We should find a better solution that this, but it's fine for now!

Comment (Edholm) [inline frontend/src/routes/SingleShowing.js]: yup, that was my thought as well. I'm thinking fancy UserCards

File: backend/src/main/kotlin/rocks/didit/sefilm/domain/Swish.kt (hunk 1 above)
Comment (Edholm) [inline backend/src/main/kotlin/rocks/didit/sefilm/domain/Swish.kt]: Yup, we can use Guavas implementation that "correctly" encodes. That is how I do it in swosh.me

**Pull request merged**